src/transformers/models/idefics/modeling_tf_idefics.py

```diff
@@ -0,0 +1,1655 @@
+# coding=utf-8
+# Copyright 2022 EleutherAI and the HuggingFace Inc. team. All rights reserved.
+#
+# This code is based on EleutherAI's GPT-NeoX library and the GPT-NeoX
+# and OPT implementations in this library. It has been modified from its
+# original forms to accommodate minor architectural differences compared
+# to GPT-NeoX and OPT used by the Meta AI team that trained the model.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" TF 2.0 Idefics model. """
+from dataclasses import dataclass
+from typing import List, Optional, Tuple, Union
+
+import tensorflow as tf
+
+from ... import TFPreTrainedModel
+from ...activations_tf import get_tf_activation
+from ...modeling_tf_outputs import ModelOutput
+from ...modeling_utils import PretrainedConfig
```

Comment:
Just realized there's a torch import issue here too - hopefully this finally resolves the tests!

```suggestion
from ...modeling_tf_utils import PretrainedConfig
```

Reply:
thank you for catching this @Rocketknight1. So far I have only been fixing the test failures from running `!py.test -vv tests/models/idefics/test_modeling_tf_idefics.py`. If we still have more test failures from the CI after above change, I will look into them